zstacklib/zstacklib/utils/qmp.py

```
@@ -84,3 +136,177 @@ def qmp_subcmd(qemu_version, s_cmd):
+        """
+        Build and send a QMP command to the monitor, report errors if any
+        """
+        if kwds:
+            keys_to_replace = [k for k in kwds if "_" in k]
+            for old_key in keys_to_replace:
+                new_key = old_key.replace("_", "-")
+                kwds[new_key] = kwds[old_key]
+                del kwds[old_key]
+        ret = self._cmd(cmd, kwds)
+        if "error" in ret:
+            raise Exception(ret['error']['desc'])
```

Ruff (0.14.5)

Create your own exception

(TRY002)